src/3rdparty/kissfft/test/Makefile

```
@@ -1,101 +1,188 @@
+#
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)